sound/soc/sof/intel/hda-sdw-bpt.c

```diff
@@ -0,0 +1,352 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2024 Intel Corporation. All rights reserved.
```

Comment:
also remove "All rights reserved", not aligned with Intel recommendations.

Reply:
done